### DISCUSSION TITLE
Any plans for a new PostgreSQL cartridge build for Windows?

### DISCUSSION THREAD

**Discussion by kienerj** [rdkit/rdkit#4073] [Q&A]
I'm wondering why there aren't any new versions of the cartridge available for windows:

![image](https://user-images.githubusercontent.com/1567309/115684769-2beeae00-a347-11eb-98e5-4e2a9449f07d.png)

Is there a plan to continue to make them in the future or are these windows builds now considered "out-of-scope"?

Thanks for clarification.

**Comment by greglandrum** [comment]
@kienerj Doing this stuff on windows isn't particularly easy and the last few times I've tried to do a windows build things have failed.
So I more or less gave up.

I can try and make some time this weekend to see if I can get a build working again.

**Comment by LucMiaz** [comment]
I know this is an old thread, but I was also looking for ways to install a newer version of the cartridge for newer versions of PostgreSQL. After a few tries I managed to build rdkit from source with the PostgreSQL cartridge on Windows mostly without trouble. I wrote down the steps I followed (gathered largely from the discussions here) as well as the work-around to some issues I had during the install in a [Medium post](https://medium.com/@luc.miaz/installing-rdkit-postgresql-cartridge-on-windows-73d632774ebe?source=friends_link&sk=0672b4e3d40eb2f9caae6d086182081f) in case that can be useful to someone.

**Reply by greglandrum** [reply]
Cartridge builds for windows are now also available from conda-forge:
https://anaconda.org/conda-forge/rdkit-postgresql

Many thanks to @skearnes for getting this working
